ci/images/ci-runner/hack/bin/destroy-cluster.sh

```diff
@@ -9,6 +9,18 @@ SCRIPT_DIR="$(
   pwd
 )"
 
+# Give developers 15mins to connect to a pod and remove the file
+# if they want to investigate the failure
+if [ -e "$PWD/destroy-cluster.txt" ]; then
+    sleep 900
+    if [ -e "$PWD/destroy-cluster.txt" ]; then
+      echo "Failure is not being investigated, cluster will be destroyed."
+    else
+      echo "Failure under investigation, cluster will not be destroyed."
```

Comment:
When the cluster is reserved , how long it will be destroyed depends on the MAX_DURATION_MINS defined in [destroy-clusters.sh](https://github.com/openshift-pipelines/pipeline-service/blob/main/ci/images/ci-runner/hack/bin/destroy-clusters.sh). Do we need to extend the MAX_DURATION_MINS bigger?

Reply:
I think 2hrs is long enough for the moment. We can change it later if we get feedback that it's too short.